package.json

```diff
@@ -14,25 +14,29 @@
     "@angular/core": "~6.0.9",
     "@angular/forms": "~6.0.9",
     "@angular/http": "~6.0.9",
+    "@angular/material": "~6.3.3",
     "@angular/platform-browser": "~6.0.9",
     "@angular/router": "~6.0.9",
     "carbon-components": "~8.23.0",
-    "primeng": "~6.0.1"
+    "primeng": "~6.1.7"
```

Comment:
ugraded primeng, previous version did not had keyboard navigation support, this one had

Reply:
Tree navigation there are issues, but its all default behavior, 
i see it breaking when folder is empty, and we press enter or right arrow key
which we can iron out, but its a good start, thanks!

including those changes will make PR even bigger, so will address them separately, Thanks!